src/main/java/sirius/biz/model/ContactData.java

```diff
@@ -108,7 +119,8 @@ public class ContactData extends Composite {
      * @throws HandledException in case of an invalid phone number
      */
     public void verifyPhoneNumber() {
-        if (Strings.isFilled(phone) && !VALID_PHONE_NUMBER.matcher(phone).matches()) {
+        if (Strings.isFilled(phone) && phone.length() < MAX_PHONE_NUMBER_LENGTH && !VALID_PHONE_NUMBER.matcher(phone)
```

Comment:
So an entered phone number that is 256 or longer is automatically valid?

Reply:
The check has ben overhauled and now uses a normal regex with possesive quantifiers.